app/src/main/java/com/example/taskintern/view/DetailActivitys.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.taskintern.view
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.navigation.fragment.navArgs
+import com.example.taskintern.R
+import com.example.taskintern.databinding.ActivityDetailActivitysBinding
+import com.example.taskintern.model.Weather
+
+class DetailActivitys : AppCompatActivity() {
+    private lateinit var binding: ActivityDetailActivitysBinding
+
```

Comment:
Fazla olan satir bosluklarini silebiliriz. Bunun icin kod formatter ktlint kullanabiliriz

Reply:
Evet, hatta ktlint plugin var, onu kullanabilirsin. Tabii önce lint nedir ne için kullanılır öğrenmelisin :) @tyagci024